src/frontend/javascript/init.js

```diff
@@ -1,3 +1,43 @@
-(function() {
-  window.GOVUKFrontend.initAll();
-})();
+// govuk-frontend has a version of this polyfill but it isn't
+// attached to globals so we cannot initialise it
+// instead this helper uses a native NodeList if browser supports it
+// else good'ol for loop
+var nodeListToArray = function (nodes, callback, scope) {
+  if (window.NodeList && Array.from) {
+    return Array.from(nodes, callback)
+  }
+
+  for (var i = 0; i < nodes.length; i++) {
+    callback.call(scope, nodes[i], i, nodes)
+  }
+}
```

Comment:
It feels like it's maybe overkill to create this function just to save ourselves two for loops 😄 - it's not even that much clearer:

```js
  nodeListToArray($buttons, function ($button) {
    new GOVUKButton($button).init();
  });

  // Versus
  for (var i = 0; i< $buttons.length, i++ {
    new GOVUKButton($buttons[i]).init();
  });
```

Reply:
yes we can do that for simplicity for now.